V1/models.py

```diff
@@ -0,0 +1,164 @@
+'''Models.py'''
```

Comment:
Give a brief description of what the modules implements and give any example of the most complex implementation that would be had to understand within the documentation.